src/main/java/Duke.java

```diff
@@ -1,10 +1,19 @@
 public class Duke {
+    /**
+     * Initializes application and starts interaction with user.
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        ConsoleInterface consoleInterface = new ConsoleInterface();
+
+        ConsoleInterface.printLogo();
+        ConsoleInterface.printLineSeparator();
+        ConsoleInterface.printGreetingMessage();
+        ConsoleInterface.printLineSeparator();
```

Comment:
Methods are clearly phrased and can be understood easily. Perhaps could combine all functions into a larger function to not show less important functions like printLineSeperator

Reply:
As above - it may be worthwhile to have a function for printing text surrounded with line separators to decrease repetition